apps/web-customer/src/pages/issue-tracking.tsx

```diff
@@ -0,0 +1,277 @@
+import React, { useState } from "react";
+import Image from "next/image";
+import IssueTrackingBG from "core-library/assets/IssueTrackingBG.png";
+import { Button, Input } from "core-library/components";
+import { useApi } from "core-library/hooks";
+import {
+  Stepper,
+  Step,
+  StepLabel,
+  StepConnector,
+  Skeleton,
+  stepConnectorClasses,
+  styled,
+} from "@mui/material";
+
+import {
+  NCLEXBlueLogo,
+  CoreZigmaLogo,
+  HelpOutlineTracking,
+  StepActive,
+  StepCircleOutline,
+} from "core-library/assets";
+
+interface IssueTracking {
+  refNo: string;
+  reportedIssueStatus: number;
+}
+
+const IssueTracking = () => {
+  const [searchReference, setSearchReference] = useState("");
+  const [issueTrackingData, setIssueTrackingData] =
+    useState<IssueTracking | null>(null);
+  const [hasSearched, setHasSearched] = useState(false);
+  const [loading, setLoading] = useState(false);
+
+  const getIssueTracking = useApi((api) =>
+    api.web.getReportIssue(searchReference)
+  );
+
```

Comment:
i believe we can achieved the loading effect through using getIssueTracking.loading
so we just need to seperate the component for the result data, and pass this so that the result will still be the same. 

```suggestion
 if(getIssueTracking.loading) return <IssueTrackingSkeletonLoader />;
```